tests/configs.py

```diff
@@ -1,13 +1,12 @@
-from __future__ import absolute_import
-from __future__ import print_function
+
 import os
 from _universum.configuration_support import Variations
 
 env_name = "virtual_universe"
 
 
 def run_virtual(cmd):
-    return ["env", "-i", "PATH=" + os.getenv("PATH"), "bash", "-c", "source {}/bin/activate; {}".format(env_name, cmd)]
+    return ["env", "-i", "PATH=" + os.getenv("PATH"), "bash", "-c", f"source {env_name}/bin/activate; {cmd}"]
```

Comment:
20:55:34  2. Processing project configs
20:55:34   |   Error: Exception while processing 'configs.py'
20:55:34   |   Traceback (most recent call last):
20:55:34   |     File "/usr/local/lib/python2.7/dist-packages/_universum/modules/launcher.py", line 273, in process_project_configs
20:55:34   |       exec(config.read(), config_globals)  # pylint: disable=exec-used
20:55:34   |     File "<string>", line 9
20:55:34   |       return ["env", "-i", "PATH=" + os.getenv("PATH"), "bash", "-c", f"source {env_name}/bin/activate; {cmd}"]
20:55:34   |                                                                                                              ^
20:55:34   |   SyntaxError: invalid syntax

Reply:
please use python3.7 instead